project/Prototypes.scala

```diff
@@ -107,13 +107,17 @@ trait Prototypes {
     topLevelDirectory in Universal := Some(application),
     concurrentRestrictions in Universal := List(Tags.limit(Tags.All, 1)),
     riffRaffPackageType := (packageBin in Universal).value,
-    riffRaffPackageName := application,
     riffRaffBuildIdentifier := System.getProperty("build.number", "DEV").replaceAll("\"",""),
     riffRaffUploadArtifactBucket := System.getenv().getOrDefault("RIFF_RAFF_ARTIFACT_BUCKET", "aws-frontend-teamcity"),
     riffRaffUploadManifestBucket := System.getenv().getOrDefault("RIFF_RAFF_BUILD_BUCKET", "aws-frontend-teamcity"),
     riffRaffArtifactPublishPath := application,
     riffRaffManifestProjectName := s"dotcom:$application",
-    artifactName := { (sv: ScalaVersion, module: ModuleID, artifact: Artifact) =>
+    riffRaffPackageName := s"dotcom:$application",
```

Comment:
this parameter will only affect the uploading of riffraff artifacts. Internal teamcity doesn't perform that task, and the archive contents are placed in a folder without the 'dotcom' prefix. So everything should stay the same.